pkg/model/combine.go

```diff
@@ -4,18 +4,18 @@ import (
 	"bytes"
 	"fmt"
 
-	"github.com/grafana/tempo/tempodb/encoding/common"
-
 	"github.com/grafana/tempo/pkg/model/trace"
 	"github.com/grafana/tempo/pkg/tempopb"
 	"github.com/pkg/errors"
 )
 
 type objectCombiner struct{}
 
-var ObjectCombiner = objectCombiner{}
+type ObjectCombiner interface {
```

Comment:
Since all of this model code is explicitly written for the "byte slice" style blocks I'm wondering if eventually this should all be consolidated under tempodb somewhere next to  the block types that use it. I suppose, however, that if we do move to a columnar format the point is moot b/c this basically all gets deleted anyway. 

Way too early to call now, but I wonder if there will be any reason to support both block styles?